Mesh/src/measurement.h

```diff
@@ -11,26 +11,34 @@
 
 ClosedCube_HDC1080 hdc1080;
 
-int measurement_memory_start = 8; // first 8 bytes are reserved
+RTC_DATA_ATTR unsigned int measurement_count = 0; // unsure if this will just stay as 0, so we need to check
 
 class Measurement
 {
 public:
+    unsigned int measurement_num;
     unsigned int moisture_percent;
     double temperature;
     double humidity;
     unsigned int light_level;
     unsigned int battery_level;
     void printMeasurement()
     {
+        Serial.println("Measurement Number: " + String(NODE_ADDRESS) + ":" + String(measurement_num));
         Serial.println("Moisture: " + String(moisture_percent) + "%");
         Serial.println("Temperature: " + String(temperature) + "F");
         Serial.println("Humidity: " + String(humidity) + "%");
         Serial.println("Light Level: " + String(light_level));
         Serial.println("Battery Level: " + String(battery_level) + "mV");
     }
+    boolean notEmpty()
+    {
+        return measurement_num < num_measurements;
+    }
 };
 
+RTC_DATA_ATTR Measurement measurements[MAX_MEASUREMENTS]; // measurements stored in RTC memory
```

Comment:
I'm assuming this allocates to the RTC memory

Reply:
should test to see if we can read back after sleep

Reply:
we can